GameDefinition/PLAYING.md

```diff
@@ -527,6 +539,90 @@ influence the speed of HP attrition and if it outweighs the benefits
 of exploration.
 
 
+Managing inventory and crafting
+-------------------------------
+
+All items belonging to your team are stored either in the character's personal 
+equipment outfit or in the shared inventory stash that is accessible by every 
+team member. 
+Items that are in the personal equipment outfit can be accessed by or 
+are beneficial for this particular character only. Stashed items can be 
+accessed by the team members (unless the character is not calm enough 
+or injured). But no one has a direct advantage from having the items 
+in the shared inventory stash.
+The stashed items are exactly stashed, they are not simply laying on
+the ground (although the stash has a particular location on the map). You
+cannot operate the items from the stash, as if these items are just on
+the ground directly. To collect an item from the stash or drop it on the ground,
+you should use commands.
+
+Obviously, if the item is beneficial (a piece of armor, weapon, or a bonus
+item), it is reasonable to put it into a particular character's outfit.
+However, you should remember that the outfit is limited to any 10 (ten)
+items only. On the contrary, if the item is not required directly (ranged
+weapon, medkits, ampoules with drugs, food, or else), it may be stored in
+the shared inventory stash.
+
+The shared inventory stash is of unlimited size. It has a physical location on
+the map and may be looted. So, it is wise to keep it safe or change 
+its location depending on the current situation. It would be a pity to lose 
+valuables or powerful projectiles by not taking care of the shared stash. 
+
+To relocate the shared inventory stash, first, you need to remove ALL 
+of the items from it anywhere you want to make it empty. Then, any stashed
+item (or a pile of items) will automatically become the new stash. The process 
+takes several turns, so you should plan shared inventory stash relocation
+in advance.
+You should keep your eye on the item management stat in the character menu
+`#`. If this stat is too low (which may be caused by an injury or low 
+calmness level), it will be difficult for the character to work with items in 
```

Comment:
This is not a bad idea, perhaps low calmness should affect item management and not directly equipment availability. However, the it's set up right now is that some rare effects just temporarily lower the item management stat (you can see the effects and how long they are going to last in the organs menu). Also, some characters (e.g., animals and some aliens or robots) innately have too low item management stat to manage items.

Calmness and HP injury have nothing to do with this (HP loss indirectly affects equipment management only in that it lowers Calm).

Reply:
Well, injury is not about the HP loss but also about being deafened or temporarily stunned (for example, by the explosion). I wanted to mention that the item managing stat may change, but not giving spoilers.

Reply:
Oh, I see now. I thought "injury" is only HP loss, but you are right in dictionary sense, though not every temporary handicap needs to be an injury in real life (e.g., flashbang may not cause an injury, despite causing a loss of combat effectiveness). Is there a good near-synonym of injury that aligns better with short loss of a capability instead of a longer lasting bodily harm?

Reply:
Perhaps, "Trauma" may work?
IMHO, "Injury" is neutral enough. I have checked synonyms briefly, but other words are more about the direct physical damage that can attribute to the HP loss.

Reply:
Well, there is a perfectly accurate term, "debuff", but it's gaming jargon and so breaks immersion. "Affliction" may be? "Impairment"? "Impediment"? Is any of these used in the manual, in-game help or the source code in general? "Injury" is not bad, but if we can be more precise, let's try.

Reply:
Probably, "debuff" is an accurate term, but I am not familiar with it. "Injury" or "trauma" IMHO are somewhat more atmospheric :)
What about "impairment"?

Reply:
Here's what I've found used currently:

```
~/r/LambdaHack$ git grep -i3 affliction
GameDefinition/PLAYING.md-- Q: Why are there two 'weakened' conditions in the organ menu of my hero?
GameDefinition/PLAYING.md-
GameDefinition/PLAYING.md-A: Each team has a different recipe for their weakness brew.
GameDefinition/PLAYING.md:Consequently, multiple affliction by the concoction from a single team
GameDefinition/PLAYING.md:prolongs a single malady, but affliction by concoctions from many teams
GameDefinition/PLAYING.md-causes concurrent ailments, with compounded effects, but independent
GameDefinition/PLAYING.md-and short durations. The benefit of the mechanism is that
GameDefinition/PLAYING.md-it's possible to tell the perpetrator team of any ailment.
~/r/LambdaHack$ git grep -i3 Impairment
~/r/LambdaHack$ git grep -i3 Impediment
~/r/LambdaHack$ git grep -i3 Injury
~/r/LambdaHack$ git grep -i3 debuff
~/r/LambdaHack$ git grep -i3 trauma
GameDefinition/Content/ItemKind.hs--- Not flashbang, because powerful bang without fragmentation is harder
GameDefinition/Content/ItemKind.hs--- to manufacture (requires an oxidizer and steel canister with holes).
GameDefinition/Content/ItemKind.hs--- The bang would also paralyze and/or lower the movement skill
GameDefinition/Content/ItemKind.hs:-- (out of balance due to ear trauma).
GameDefinition/Content/ItemKind.hs-flashBomb = fragmentationBomb
GameDefinition/Content/ItemKind.hs-  { iname    = "magnesium ribbon"  -- filled with magnesium flash powder
GameDefinition/Content/ItemKind.hs-  , iflavour = zipPlain [BrYellow]  -- avoid @BrWhite@; looks wrong in dark
```

If "affliction", "ailment" and "malady" are not to your liking, "impairment" certainly is fine. I like "trauma" a lot, too.

Reply:
For completeness, in Allure's content I've found another occurrence: 

```
GameDefinition/Content/ItemKind.hs:  , idesc    = "A vial of bright, frothing concoction. 
The best medicine that nature has to offer for wounds, ailments and mood swings."
```

Reply:
I suggest "trauma". IMHO, it suggests that a negative event can happen during the fight. It is ambiguous enough to not give spoilers directly. And it is atmospheric.
But I can change "injury" to "malady", "ailment", "impairment", or "ailment".

Reply:
Trauma it is.

Reply:
OK!